objectivec/GPBDictionary.h

```diff
@@ -60,55 +60,6 @@ NS_ASSUME_NONNULL_BEGIN
 @property(nonatomic, readonly) NSUInteger count;
 
 /**
- * @return A newly instanced and empty dictionary.
- **/
-+ (instancetype)dictionary;
```

Comment:
alloc/init works for most other classes. I don't see a need for it. If people want real short hand they can use "new"